docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: global
+title: Running Facebook Presto with Alluxio
+nickname: Presto
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Facebook Presto](https://prestodb.io/) with Alluxio, so
+that you can easily use presto to query Hive tables in Alluxio's tiered storage.
+
+# Prerequisites
+
+The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must use Java 8 Update 60 or higher (8u60+), 64-bit.
+Alluxio cluster should also be set up in accordance to these guides for either
+[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+Alluxio client will need to be compiled with the Presto specific profile. Build the entire project
+from the top level `alluxio` directory with the following command:
+
+```bash
+mvn clean package -Ppresto -DskipTests
```

Comment:
Yes, presto just need the client jar. I will check this profile whether it can work?